### PR TITLE
Iteration 6

### PR DESCRIPTION
**Pull request by aliciatay-zls**
Complete to-do list by mentor:
- [x] Improve ability to run setup scripts in sequence
- [x] Add ability to generate keywords and add to db in each parser
- [ ] Handle empty parameters in rest_api.js
- [x] Implement a CSS framework
- [ ] Add to README.md
- [x] Shift all MySQL queries in current .sql files to migration files

Extra:
- [ ] Add new sample data: a restaurant

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "parse-kfc": "node parser_kfc.js",
     "parse-mcd": "node parser_mcd.js",
     "parse-nfp": "node parser_nfp.js",
-    "setup": "npm run parse-kfc && npm run parse-mcd && npm run parse-nfp && npx knex seed:run && npm run dev",
+    "db:create": "echo Creating new DB && node create_db.js",
```
Comment (miccheng) [inline package.json]: LOL... might wanna change the `echo` to `console.log` in `create_db.js`.

**Review by miccheng — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (aliciatay-zls) [reply package.json]: > LOL... might wanna change the `echo` to `console.log` in `create_db.js`.

I was thinking that it's random too. Okay will do 😂
Edit: changed in 4a9ff22

File: package.json [hunk 2]
```diff
@@ -18,8 +18,9 @@
     "parse-kfc": "node parser_kfc.js",
     "parse-mcd": "node parser_mcd.js",
     "parse-nfp": "node parser_nfp.js",
-    "db:create": "echo Creating new DB && node create_db.js",
-    "db:setup": "npm run parse-kfc && npm run parse-mcd && npm run parse-nfp && npx knex seed:run && npm run dev",
+    "db:create": "node create_db.js",
+    "db:setup": "npx knex migrate:latest",
```
Comment (miccheng) [inline package.json]: I wonder if you might already have the Knex CLI in the node_modules folder. Check the `node_modules/bin` folder. If it's there, there's no need to use npx.

**Review by miccheng — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (aliciatay-zls) [reply package.json]: Got it, thank you Mr Cheng.
Edit: corrected in b5dc3e0 and f13da99

**Review by miccheng — APPROVED** [review]
LGTM

**Pull request merged**